packages/aws-cdk/bin/cdk.ts

```diff
@@ -66,14 +70,14 @@ async function parseCommandLineArguments() {
     .option('staging', { type: 'boolean', desc: 'Copy assets to the output directory (use --no-staging to disable, needed for local debugging the source files with SAM CLI)', default: true })
     .option('output', { type: 'string', alias: 'o', desc: 'Emits the synthesized cloud assembly into a directory (default: cdk.out)', requiresArg: true })
     .option('no-color', { type: 'boolean', desc: 'Removes colors and other style from console output', default: false })
-    .command(['list [STACKS..]', 'ls [STACKS..]'], 'Lists all stacks in the app', yargs => yargs
+    .command(['list [STACKS..]', 'ls [STACKS..]'], 'Lists all stacks in the app', (yargs: any) => yargs
```

Comment:
You can still `import type * as yargs from 'yargs'`, I believe.

Reply:
Nope. `yargs` is used as a value here: 

https://github.com/aws/aws-cdk/blob/ea7802b4b5ea4781ed884fc8d330173b192d569d/packages/aws-cdk/bin/cdk.ts#L51-L52